pkg/scheduler/framework/plugins/defaultpreemption/default_preemption.go

```diff
@@ -180,11 +261,22 @@ func FindCandidates(ctx context.Context, cs kubernetes.Interface, state *framewo
 		}
 		klog.Infof("%v potential nodes for preemption, first %v are: %v", len(potentialNodes), len(sample), sample)
 	}
+
 	pdbs, err := getPodDisruptionBudgets(pdbLister)
 	if err != nil {
 		return nil, err
 	}
-	return dryRunPreemption(ctx, ph, state, pod, potentialNodes, pdbs), nil
+
+	fco := &FindCandidatesOptions{}
+	for _, opt := range opts {
+		opt(fco)
+	}
+	offset, numCandidates := fco.updateAndReturnOffset(int32(len(potentialNodes)))
+	if klog.V(5).Enabled() {
+		klog.Infof("dry running preemption to find ~%d candidates (starting offset: %d)", numCandidates, offset)
```

Comment:
How about move the log line 258~262 to here, then we can get the node name also.

Reply:
Done.